nomad/rust-integration-tests.nomad

```diff
@@ -159,6 +154,11 @@ job "rust-integration-tests" {
               destination_name = "event-source"
               local_bind_port  = 1005
             }
+
+            upstreams {
+              destination_name = "web-ui"
+              local_bind_port  = 1007
```

Comment:
noti mportant, but why not 1006

Reply:
Ah, no good reason: originally I had another upstream using 1006, I just hadn't updated this one after removing the other. I'll fix that to use 1006.